SConstruct

```diff
@@ -1,7 +1,26 @@
 import os
+import subprocess
 
 # Load environment variables, including some that should be renamed.
 env = Environment(ENV = os.environ)
+
+msys = (env['PLATFORM'] == 'msys')
+if msys:
+	# Expecting mingw-w64 set up through MSYSTEM.
+	env = Environment(ENV = os.environ, tools=['mingw'])
```

Comment:
`env.Append(tools = ['mingw'])`?

(Since `env` is initialized with `os.environ already`)

Reply:
Setting `tools` during initialization has other side effects so `env` must be replaced.

Reply:
Sounds good :+1: